src/macros.rs

```diff
@@ -824,25 +824,39 @@ macro_rules! clap_app {
 }
 
 macro_rules! impl_settings {
-    ($n:ident, $($v:ident => $c:path),+) => {
-        pub fn set(&mut self, s: $n) {
-            match s {
-                $($n::$v => self.0.insert($c)),+
+    ($settings:ident, $flags:ident,
+        $( $setting:ident($str:expr) => $flag:path ),+
+    ) => {
+        impl $flags {
+            pub fn set(&mut self, s: $settings) {
+                match s {
+                    $($settings::$setting => self.0.insert($flag)),*
```

Comment:
Shouldn't the end be `+` instead of `*`? Similarly below.

Reply:
Again, doesn't change anything, but if you insist...

Reply:
Nope, `*` is correct here